### PR TITLE
Add 'Nylas N1' version 0.3.25-c174641.

### PR DESCRIPTION
**Pull request by weijianwen**
'Nylas N1' is a extensible, open source mail client.

### CODE REVIEW

File: Casks/nylas-n1.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'nylas-n1' do
+  version '0.3.25-c174641'
+  sha256 '48f77c8b1a0957b4edafe201362658e8798339e29b4b639cc1cd11404daab234'
+
+  url 'https://edgehill.s3-us-west-2.amazonaws.com/0.3.25-c174641/darwin/x64/N1.dmg'
```
Comment (vitorgalvao) [inline Casks/nylas-n1.rb]: Needs a `# amazonaws.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (vitorgalvao) [inline Casks/nylas-n1.rb]: Also, should be `url "https://edgehill.s3-us-west-2.amazonaws.com/#{version}/darwin/x64/N1.dmg"`.

**Comment by weijianwen** [comment]
@vitorgalvao thoes issues are fixed in my new commit. "nalys-n1" is ready for merging now.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as d4caf4f2e5af6905cbc24058ca753189d8e2a72b to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**